### PR TITLE
Reformatting Getting-Started

### PR DESCRIPTION
**Pull request by fshmcallister**
Our Getting-Started page was great for people who already had a good grasp/ knew about security; but wasn't necessarily newbie-friendly. I've restructured the page to take them through installation faster and leaving them to read on if they're interested in learning more. Further restructures to be added to tutorials in a similar fashion.

Notes appreciated @AlEsmail @vpetersson

### CODE REVIEW

File: documentation/getting-started.md [hunk 1]
```diff
@@ -18,35 +18,21 @@ We provide this in a number of ways:
...
-
-As a developer, you will likely be accessing a large range of internet/cloud services that require basic HTTP authentication or an API key. In most instances, such credentials are hard coded into applications and can be easily read with access to said applications. WoTT introduces credential management so that you can remotely download the necessary credentials onto your device as a JSON file. This removes the need for hard coded keys and logins from your application.
-
-We do this via our Dashboard which you can use to monitor your devices. The Dashboard gives your device a trust score (based on how secure it is) as well as giving you the means to add and manage credentials for whatever purpose you need them for.
+First, let's get the WoTT agent set up.
 
 ## Installing the WoTT agent and claiming your device
 
-First, create an account for the [WoTT Dashboard](https://dash.wott.io) and login.
+You will need to register for an account on the [WoTT Dashboard](https://dash.wott.io) and login. You can manage the credentials of up to 20 devices for free. For additional features, we offer more comprehensive packages. See our [prices]({{site.url}}/pricing) for more information.
```
Comment (AlEsmail) [inline documentation/getting-started.md]: Remove the info about 20 devices. We aren't sure on this point and it creates complexity when we change it.

File: documentation/getting-started.md [hunk 2]
```diff
@@ -18,35 +18,21 @@ We provide this in a number of ways:
 * Cryptographic identity (x509 certificate issued from our CA)
 * Enabling the removal of hard coded credentials from web applications and firmware
 
-You can access all these features by installing the [WoTT Agent](https://github.com/WoTTsecurity/agent) and using the [WoTT Dashboard](https://dash.wott.io) to manage your devices through a simple and user-friendly interface. We'll walk you through step-by-step in setting up and installing the WoTT agent so you can start to implement hassle-free security on your IoT or edge devices.
```
Comment (AlEsmail) [inline documentation/getting-started.md]: Remove link to github. send people to the registration page in both cases. We want to drive sign ups as the call to action.

**Comment by vpetersson** [comment]
@fshmcallister is `google-core-iot-and-hat.md` work in progress?

File: documentation/getting-started.md [hunk 3]
```diff
@@ -10,47 +10,31 @@ redirect_from:
...
-
-As a developer, you will likely be accessing a large range of internet/cloud services that require basic HTTP authentication or an API key. In most instances, such credentials are hard coded into applications and can be easily read with access to said applications. WoTT introduces credential management so that you can remotely download the necessary credentials onto your device as a JSON file. This removes the need for hard coded keys and logins from your application.
-
-We do this via our Dashboard which you can use to monitor your devices. The Dashboard gives your device a trust score (based on how secure it is) as well as giving you the means to add and manage credentials for whatever purpose you need them for.
+We'll provide a step-by-step walkthrough of set-up and installation of the WoTT agent. It's hassle-free and simple to implement.
 
 ## Installing the WoTT agent and claiming your device
```
Comment (vpetersson) [inline documentation/getting-started.md]: `s/device/node`

**Comment by fshmcallister** [comment]
> @fshmcallister is `google-core-iot-and-hat.md` work in progress?

Good spot - yes, for some reason switching branches didn't remove the file? I'll just take it out of the pull request

File: documentation/getting-started.md (hunk 3 above)
Comment (fshmcallister) [reply documentation/getting-started.md]: I'm confused by this

Comment (vpetersson) [reply documentation/getting-started.md]: @fshmcallister you not fluent in regular expression? ;)

Replace device with node in the article

Comment (fshmcallister) [reply documentation/getting-started.md]: bully :P

Comment (fshmcallister) [reply documentation/getting-started.md]: how come node though?

Comment (vpetersson) [reply documentation/getting-started.md]: We're tweaking the messages to be less IoT oriented and more towards servers. Node is a nice catch-all. We're toying with words like 'node', 'endpoint', 'system'. Happy to get you input if you can think of any other nice catch-all.

Comment (fshmcallister) [reply documentation/getting-started.md]: Nope no problem! Let me reword a little more then since getting started is still quite IoT focused

**Comment by vpetersson** [comment]
Thanks @fshmcallister!

**Pull request merged**